crates/pallet-humanode-session/src/lib.rs

```diff
@@ -72,6 +74,67 @@ pub mod pallet {
     #[pallet::storage]
     pub type CurrentSessionIndex<T: Config> = StorageValue<_, SessionIndex, OptionQuery>;
 
+    /// A list of all banned accounts that can't be validators in the network.
+    #[pallet::storage]
+    pub type BannedAccounts<T: Config> =
+        StorageValue<_, BoundedVec<T::AccountId, T::MaxBannedAccounts>, ValueQuery>;
+
+    /// Possible errors.
+    #[pallet::error]
+    pub enum Error<T> {
+        /// Attempt to disable bootnode to be a validator.
+        AttemptToDisableBootnode,
+        /// Failed to convert `T::AccountId` to `T::ValidatorId`.
+        AccountIdToValidatorId,
+        /// The provided account could not be found in current validators list or it was already
+        /// disabled.
+        AccountIsNotValidator,
+        /// The account is already banned.
+        AccountIsAlreadyBanned,
+        /// The BannedAccounts storage has reached the limit as BoundedVec.
+        TooManyBannedAccounts,
+    }
+
+    #[pallet::call]
+    impl<T: Config> Pallet<T> {
+        /// Kick validator based on provided account id.
+        #[pallet::call_index(0)]
+        #[pallet::weight(0)]
+        pub fn kick(origin: OriginFor<T>, account_id: T::AccountId) -> DispatchResult {
+            ensure_root(origin)?;
```

Comment:
Can we make this generic, so that we can tweak this check at the runtime level?

Reply:
So, it can be executed not just as sudo call?

Reply:
Kick call is removed.